### PR TITLE
202402blogs

### PR DESCRIPTION
**Pull request by HelmerDenDekker**
Check!!

### CODE REVIEW

File: docs/assets/images/c4model/zoomlevel.svg
Comment (HelmerDenDekker) [inline docs/assets/images/c4model/zoomlevel.svg:1]: Revert

File: docs/assets/images/domaindrivendesign/domaindrivendesign.svg
Comment (HelmerDenDekker) [inline docs/assets/images/domaindrivendesign/domaindrivendesign.svg:1]: Revert

File: docs/assets/images/domaindrivendesign/domaindrivendesignlayers.svg
Comment (HelmerDenDekker) [inline docs/assets/images/domaindrivendesign/domaindrivendesignlayers.svg:1]: Revert

File: docs/assets/images/domaindrivendesign/domaindrivendesignmodel.svg
Comment (HelmerDenDekker) [inline docs/assets/images/domaindrivendesign/domaindrivendesignmodel.svg:1]: Revert

File: docs/assets/images/internet/internetdeviceserver.svg
Comment (HelmerDenDekker) [inline docs/assets/images/internet/internetdeviceserver.svg:1]: Revert

File: docs/assets/images/internet/internetfromto.svg
Comment (HelmerDenDekker) [inline docs/assets/images/internet/internetfromto.svg:1]: Revert

File: docs/assets/images/kafka/kafka.svg
Comment (HelmerDenDekker) [inline docs/assets/images/kafka/kafka.svg:1]: Revert

File: docs/assets/images/kafka/messaging.svg
Comment (HelmerDenDekker) [inline docs/assets/images/kafka/messaging.svg:1]: Revert

File: docs/assets/images/nunit/testexecnunit.svg
Comment (HelmerDenDekker) [inline docs/assets/images/nunit/testexecnunit.svg:1]: Revert

File: docs/assets/images/rx/observerpattern.svg
Comment (HelmerDenDekker) [inline docs/assets/images/rx/observerpattern.svg:1]: Revert

File: docs/assets/images/saml/saml.svg
Comment (HelmerDenDekker) [inline docs/assets/images/saml/saml.svg:1]: Revert

**Review by TeamXerbutri — APPROVED** [review]

**Pull request merged**